### PR TITLE
Update the feed tab style

### PR DESCRIPTION
**Pull request by Teraskull**
This PR contains the following changes:

* Use the new GitHub style for tabs for consistency.
* Make the feed width a bit smaller to remove white-space.

image
image

This improves user experience by showing hover animations.

### CODE REVIEW

File: old-feed.user.js [hunk 1]
```diff
@@ -14,15 +14,19 @@
...
     const columnContainer = document.querySelector(".feed-content");
     columnContainer.classList.remove("flex-justify-center");
-    columnContainer.style.maxWidth="100vw";
+    columnContainer.style.maxWidth="unset";
     const feedColumn = columnContainer.querySelector(".feed-main");
-    feedColumn.style.maxWidth="100vw";
+    feedColumn.style.maxWidth="100%";
+    const sidebar = document.querySelector('.feed-right-sidebar');
+    sidebar.style.maxWidth = "unset";
+    sidebar.style.width = "900px";
```
Comment (Gerrit0) [inline old-feed.user.js:29]: Why this change? With a wide monitor, the left and right sidebars no longer match in width... looks weird to me.

File: old-feed.user.js [hunk 2]
```diff
@@ -36,23 +40,46 @@
...
     const picker = document.createElement("div");
-    picker.classList.add("color-border-default");
-    picker.style = "border-bottom: 1px solid; margin-bottom: 1em";
     news.insertBefore(picker, feedContainer);
-    picker.innerHTML = `<button class="Button" data-show="following">Following</button><button class="Button" data-show="forYou">For you</button><span style="float:right">Loading...</span>`;
-    const loadingIndicator = picker.querySelector("span");
+    picker.innerHTML = `
+        <div class="mb-3">
+            <nav class="overflow-hidden UnderlineNav">
+                <ul class="UnderlineNav-body">
```
Comment (Gerrit0) [inline old-feed.user.js:47]: How'd you even find these classes? I poked around for a while before giving up and making my own...

File: old-feed.user.js (hunk 1 above)
Comment (Teraskull) [reply old-feed.user.js:29]: You can skip this change if it looks bad on wide monitors. I tested only on one screen.

File: old-feed.user.js (hunk 2 above)
Comment (Teraskull) [reply old-feed.user.js:47]: It's from the repo code page:

image

I just removed all unneeded classes and icons and left the bare minimum to keep the same functionality.

**Comment by Gerrit0** [comment]
Thanks!
